src/index.ts

```diff
@@ -453,7 +454,7 @@ export function findByModel(model: string) {
 
 // Can be used to handle events for devices which are not fully paired yet (no modelID).
 // Example usecase: https://github.com/Koenkk/zigbee2mqtt/issues/2399#issuecomment-570583325
-export async function onEvent(type: OnEventType, data: OnEventData, device: Zh.Device) {
+export async function onEvent(type: OnEventType, data: OnEventData, device: Zh.Device, meta: Fz.Meta) {
```

Comment:
I think it's cleaner to make a new meta, `OnEvent.Meta`

Reply:
added. right?